launch_xml/launch_xml/entity.py

```diff
@@ -91,7 +91,7 @@ def get_attr(
         If coercion fails, `ValueError` will be raised.
         """
         attr_error = AttributeError(
-            'Attribute {} of type {} not found in Entity {}'.format(
+            'Attribute `{}` of type `{}` not found in Entity `{}`'.format(
```

Comment:
```suggestion
            "Attribute '{}' of type '{}' not found in Entity '{}'".format(
```

I would prefer using single quotes (`'`) instead of ticks, like here: https://github.com/ros2/launch/blob/cb39d92dd99d821f0aa59b40895303c5b772790b/launch/launch/event_handler.py#L83.

The same goes for the other locations.

Reply:
No problem, I also prefer them. I just did it like the other error messages in the same file.

Reply:
Oh, you're right. I believe when I made the comment I looked elsewhere in the repository and saw the single quotes in use. It'd be nice to have this more consistent within and across files.